pyigm/abssys/dla.py

```diff
@@ -70,7 +72,7 @@ def from_datfile(cls, dat_file, tree=None, **kwargs):
 
         # QSO keys
         slf.qso = slf._datdict['QSO name']
-        slf.zqso = float(slf._datdict['QSO zem'])
+        slf.zem = float(slf._datdict['QSO zem'])
```

Comment:
It's historical, from the .dat files.  e.g., 

J1134+5742                                                  ! QSO name
11:34:18.96                                                 ! RA (2000)
+57:42:04.7                                                 ! DEC (2000)
 3.522000                                                   ! QSO zem
 2                                                          ! flg_QSOmag
18.200001                                                   ! QSO Mag
 3                                                          ! QSO Survey